### PR TITLE
Design Iteration

### PR DESCRIPTION
**Pull request by teich**
Implement proposed design

[Sketch link](https://www.sketch.com/s/9c476952-055e-4919-b474-520fe563bff8/a/ZVWWxAG#Inspect)

Guide

### CODE REVIEW

File: config.yaml [hunk 1]
```diff
@@ -24,7 +24,7 @@ markup:
   highlight:
     codeFences: true
     noClasses: false
-    linenos: true
+    linenos: false
```
Comment (plumbis) [inline config.yaml:27]: (I don't have strong opinions here) why do we want to remove line numbers? This also removes the ability to link to a specific lines in an output. It looks cleaner, but it makes it less user friendly.

File: themes/upbound/assets/scss/bootstrap/_variables.scss [hunk 2]
```diff
@@ -630,11 +630,11 @@ $line-height-base:            1.5 !default;
...
-$h1-font-size:                $font-size-base * 2.5 !default;
-$h2-font-size:                $font-size-base * 2 !default;
-$h3-font-size:                $font-size-base * 1.75 !default;
-$h4-font-size:                $font-size-base * 1.5 !default;
-$h5-font-size:                $font-size-base * 1.25 !default;
+$h1-font-size:                $font-size-base * 2 !default;
+$h2-font-size:                $font-size-base * 1.5 !default;
+$h3-font-size:                $font-size-base * 1.25 !default;
+$h4-font-size:                $font-size-base * 1 !default;
+$h5-font-size:                $font-size-base * 1 !default;
```
Comment (plumbis) [inline themes/upbound/assets/scss/bootstrap/_variables.scss:637]: This makes these headings not stand out. [Example](https://docs-git-fork-teich-designv3-upboundio.vercel.app/concepts/managed-control-planes/#configure-crossplane-providers-on-your-mcp)

I'm okay with it, just making sure we're aware.

File: themes/upbound/assets/scss/_hints.scss [hunk 3]
```diff
@@ -78,28 +82,22 @@ $hint-note-background:          rgba(0,176,255,.1);
...
+    //     border-left-color: #{$hint-tip};
 
-        >.admonition-title {
-            background-color: #{$hint-tip-background};
-        }
-    }
+    //     >.admonition-title {
+    //         background-color: #{$hint-tip-background};
+    //     }
+    // }
```
Comment (plumbis) [inline themes/upbound/assets/scss/_hints.scss:91]: I like the new call out styles. Only comment is that [tips](https://docs-git-fork-teich-designv3-upboundio.vercel.app/contribute/features/#hints-and-alert-boxes) are now grey, I think we should make them a "positive" color.

File: themes/upbound/assets/scss/_hints.scss [hunk 4]
```diff
@@ -68,7 +69,10 @@ $hint-note-background:          rgba(0,176,255,.1);
...
 
 .admonition {
-    border-left: .3rem solid;
+    border-radius: 0.375rem;
+    border: 1px solid rgba(205,205,211, 1);
+    opacity: 1;
+    background-color: rgba(248,249,249, 1);
 
     &.warning {
         border-left-color: #{$hint-warning};
```
Comment (plumbis) [inline themes/upbound/assets/scss/_hints.scss:78]: Warnings are styled differently now than the other boxes by maintaining their title background bar, not sure if that was on purpose.

**Review by plumbis — APPROVED** [review]
Few things to make sure we're aware are changing, but otherwise looks great. Approved!

File: themes/upbound/assets/scss/_hints.scss (hunk 3 above)
Comment (cwilhit) [reply themes/upbound/assets/scss/_hints.scss:91]: @shanecmiller23 feedback above ^. I tend to agree but you are the design expert.

Comment (shanecmiller23) [reply themes/upbound/assets/scss/_hints.scss:91]: I don't feel strongly on this. I'd say gray works for now but I will tinker with this after this commit

File: config.yaml (hunk 1 above)
Comment (shanecmiller23) [reply config.yaml:27]: I looked through a few Docs references that were omitting the line numbers. If we can find someone else including these, I'm happy to include them

**Pull request merged**